index.html

```diff
@@ -1255,6 +1239,231 @@ <h3>
+          corresponding <code><dfn data-export="" data-dfn-for=
+          "manifest">*_localized</dfn></code> member, where `*` represents the
+          member name.
+        </p>
+        <aside class="example">
+          <p>
+            For example, the [=manifest/name=] member is a [=localizable
+            member=], and `name_localized` is its corresponding
+            [=manifest/*_localized=] member.
+          </p>
+        </aside>
+        <p>
+          A <dfn>language map</dfn> is an [=ordered map=] whose key is a
+          [=language tag=] and whose value is a [=localized value=]. The
+          <dfn>localized value</dfn> represents the value applied for the
+          [=language tag=] specified as the key.
+        </p>
+        <aside class="example" title="Localizing the application name">
+          <pre class="json">
+            {
+              "lang": "en-US",
+              "dir": "ltr",
+              "name": "Color Picker",
+              "name_localized": {
+                "de":    "Farbwähler",
+                "en":    {"value": "Color Picker"},
+                "en-GB": {"value": "Colour Picker", "dir": "ltr"},
+                "fr":    {"value": "Sélecteur de Couleur", "lang": "fr-CA", "dir": "ltr"},
+                "ar":    {"value": "منتقي الألوان", "dir": "rtl"}
+              }
+            }
+          </pre>
+        </aside>
+        <p>
+          [=manifest/*_localized=] members contain a [=language map=] that
+          defines [=localized values=] for the given [=localizable member=] in
+          the application. The user agent SHOULD use the user's localization
+          settings to select the [=localized value=] whose [=language tag=] key best
+          matches the user's preference. When no such [=localized value=] is
+          available, the default representation is used.
+        </p>
+        <section>
+          <h3>
+            Localizing text resources
+          </h3>
+          <p>
+            For [=manifest/*_localized=] members that accept a [=string=] as
+            their value, the [=localized value=] can either be a plain
+            localized [=string=] or an [=ordered map=] containing the following
+            members:
+          </p>
+          <dl>
+            <dt>
+              `value`
+            </dt>
+            <dd>
+              The localized [=string=].
+            </dd>
+            <dt>
+              `lang` (optional)
+            </dt>
+            <dd>
+              A [=language tag=].
+            </dd>
+            <dt>
+              `dir` (optional)
+            </dt>
+            <dd>
+              The [=text-direction=].
+            </dd>
+          </dl>
+          <p>
+            When a plain [=string=] is used, or when the `lang` or `dir`
+            members are omitted, `lang` falls back the [=language tag=] of the
+            [=language map=] key, and `dir` falls back to the [=manifest/dir=]
```

Comment:
I'm having trouble following this... I think you should make two paragraphs here, one for dir and one for lang. 

I was expecting the lang to fall back to to the lang defined at the root of the manifest.

Reply:
I think this means that, for instance, in `"fr": {"value": "Sélecteur de Couleur", "dir": "ltr"}` the `lang` value is `"fr"`. But I agree it could be made clearer.

I'm not sure I understand why we need both the key of the map and this `lang` value. `"fr": {"value": "Sélecteur de Couleur", "lang": "fr-CA", "dir": "ltr"}` could just be `"fr-CA": {"value": "Sélecteur de Couleur", "dir": "ltr"}` no?

I would see some value in allowing multiple values, on the other hand. This would allow having a single entry for multiple variants of a language, e.g. `"fr-others": {"value": "Sélecteur de Couleur", "lang": "fr-CA, fr-BE", "dir": "ltr"}`

Reply:
To make it even less ambiguous, I would just get rid of the language map and replace it with an array of objects:

```json
"name_localized": [
  { "lang": "de", "value": "Farbwähler" },
  { "lang": "en", "value": "Color Picker" },
  { "lang": "en-GB", "value": "Colour Picker" },
  { "lang": "fr", "value": "Sélecteur de Couleur" },
  { "lang": "fr-CA, fr-BE", "value": "Sélecteur des Couleurs" },
  { "lang": "ar", "value": "منتقي الألوان", "dir": "rtl" }
}
```

Reply:
@jcayzac @marcoscaceres This is needed in (probably rare) cases where you want the value in language A applied to locale B, for example, so that assistive technology can use the right voice for pronunciation.

I think a good example is if a brand is localized differently across countries ([actual example for such a brand](https://en.wikipedia.org/wiki/Just_Eat_Takeaway.com#Operations)):

```json
{
  "name": "Thuisbezorgd.nl",
  "lang": "nl-NL",
  "name_localized": {
    "de-DE": "Lieferando",
    "de-CH": { "lang": "en", "value": "Just eat" },
    "ro": { "lang": "en", "value": "Takeaway.com" }
  }
}
```

Or if you have a bilingual app and want to offer a “change language” shortcut in the target language:

```json
"name_localized": {
  "en": { "lang": "fr", "value": "Changer de langue" },
  "fr": { "lang": "en", "value": "Change language" }
}
```

This is why
1. we need a language map with a language tag key
1. the "lang" attribute, if not specified, must fall back to the language tag specified in the key

Reply:
@marcoscaceres I split this into [two paragraphs](https://pr-preview.s3.amazonaws.com/w3c/manifest/pull/1101.html#localizing-text-resources:~:text=When%20a%20string%20is%20used%2C%20or%20when%20the%20dir%20member%20of%20the%20localized%20text%20resource%20is%20missing%2C%20the%20default%20direction%20(dir%20member%20of%20the%20manifest)%20is%20applied.). Does that help?